firmware/applications/SignBuddy/imu.c

```diff
@@ -0,0 +1,70 @@
+#include "imu.h"
+
+#include "board.h"
+#include "common.h"
+#include "i2c.h"
+#include "logger.h"
+
+#define PROCESS_PERIOD_MS    1000
+
+typedef struct {
+  uint32_t last_ticks;
+  i2c_t    i2c_instance;
+} state_t;
+
+static state_t s;
+
+static void hw_init(void)
+{
+  LL_IOP_GRP1_EnableClock(LL_IOP_GRP1_PERIPH_GPIOB);
+  LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_I2C1);
+
+  LL_GPIO_InitTypeDef gpio_config = { 0 };
+  gpio_config.Pin = LL_GPIO_PIN_6 | LL_GPIO_PIN_7;
```

Comment:
Abstract pins and alternate function for consistency with other modules

Reply:
yeah I forgot to make that an issue.. 

I'll just do it

Reply:
Thanks